src/wily/commands/diff.py

```diff
@@ -145,12 +145,12 @@ def diff(config, files, metrics, changes_only=True, detail=True, revision=None):
                         )
                     )
                 else:
-                    metrics_data.append("{0:n} -> {1:n}".format(current, new))
+                    metrics_data.append(f"{current:n} -> {new:n}")
```

Comment:
Does this need to be:
```suggestion
                    metrics_data.append(f"{current:{n}} -> {new:{n}}")
```